server/README.md

```diff
@@ -5,14 +5,25 @@ Server application built with Node.js
 ## Major Dependencies
 
 - Express
+- Mongoose
+- Joi
 
 ## Getting Started
 
+#### Get set up with Mongo
+
+- Create user/password in RayVolution cluster
+- Whitelist your IP address
+- Copy connection string
+  - Recommended: install MongoDB Compass, connect to cluster using connection string
+- Create your test database
+
 #### Add .env file in root folder:
 
 ```
-PORT = 4000
-DATABASE_CONNECTION_STRING = <your-mongodb-connection-string-here>
+PORT=4000
+DATABASE_CONNECTION_STRING=<your-mongodb-connection-string-here>
+MY_TEST_DB=<your-test-db-name>
```

Comment:
I think we can simplify slightly and just specify the db name in the connection string. I don't mind keeping separate though if you think it's clearer.

Eg
`mongodb+srv://<username>:<password>@abcde.mongodb.net/myDatabase`
https://www.mongodb.com/resources/products/fundamentals/mongodb-connection-string